controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -331,7 +335,7 @@ func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 					}}
 				},
 			),
-			builder.WithPredicates(prometheusPredicate),
+			builder.WithPredicates(predicate.GenerationChangedPredicate{}, prometheusPredicate),
```

Comment:
if possible, let's try to be consistent, could you pls move the predicate to end like being done for all other watches?

Reply:
removed it.